cli/src/commands/explore/mod.rs

```diff
@@ -1,159 +1,223 @@
-use crate::utils::{print_first_paragraph_with_indent, validate_url_security, FileOrUrl};
-use clap::{Parser as ClapParser, Subcommand};
-
-use std::fmt::Write;
-use std::write;
-
+use crate::utils::validate_url_security;
+use crate::utils::FileOrUrl;
+use clap::{command, Parser, Subcommand};
 use codec::Decode;
 use color_eyre::eyre::eyre;
-
-use crate::commands::explore::calls::{explore_calls, CallsSubcommand};
-use crate::commands::explore::constants::{explore_constants, ConstantsSubcommand};
-use crate::commands::explore::storage::{explore_storage, StorageSubcommand};
+use color_eyre::owo_colors::OwoColorize;
+use indoc::writedoc;
+use std::fmt::Write;
+use std::write;
 
 use subxt::Metadata;
 
-mod calls;
-mod constants;
-mod storage;
+use self::pallets::PalletSubcommand;
+
+mod pallets;
+mod runtime_apis;
 
 /// Explore pallets, calls, call parameters, storage entries and constants. Also allows for creating (unsigned) extrinsics.
 ///
 /// # Example
 ///
-/// ## Pallets
-///
-/// Show the pallets that are available:
+/// Show the pallets and runtime apis that are available:
 /// ```text
 /// subxt explore --file=polkadot_metadata.scale
 /// ```
 ///
-/// ## Calls
+/// ## Pallets
+///
+/// each pallet has `calls`, `constants`, `storage` and `events` that can be explored.
+///
+/// ### Calls
 ///
 /// Show the calls in a pallet:
 /// ```text
-/// subxt explore Balances calls
+/// subxt explore pallet Balances calls
 /// ```
 /// Show the call parameters a call expects:
 /// ```text
-/// subxt explore Balances calls transfer
+/// subxt explore pallet Balances calls transfer
 /// ```
 /// Create an unsigned extrinsic from a scale value, validate it and output its hex representation
 /// ```text
-/// subxt explore Grandpa calls note_stalled { "delay": 5, "best_finalized_block_number": 5 }
+/// subxt explore pallet Grandpa calls note_stalled { "delay": 5, "best_finalized_block_number": 5 }
 /// # Encoded call data:
 /// # 0x2c0411020500000005000000
-/// subxt explore Balances calls transfer  "{ \"dest\": v\"Raw\"((255, 255, 255)), \"value\": 0 }"
+/// subxt explore pallet Balances calls transfer  "{ \"dest\": v\"Raw\"((255, 255, 255)), \"value\": 0 }"
 /// # Encoded call data:
 /// # 0x24040607020cffffff00
 /// ```
-/// ## Constants
+/// ### Constants
```

Comment:
I'd ensure a newline above and below each heading (this and below are missing one or both), and in general add newlines before and after the code blocks in this whole section (and anywhere else) just to make it more readable as plain text :)